models/quality_measures/intermediate/cqm236_controlling_blood_pressure/quality_measures__int_cqm236_numerator.sql

```diff
@@ -0,0 +1,187 @@
+    from observations
+    inner join denominator
+        on observations.patient_id = denominator.patient_id
+        and observations.observation_date between 
+            denominator.performance_period_begin and denominator.performance_period_end
+
+)
+
+, observations_with_encounters as (
+
+    select
+        observations_within_range.*
+        , case
+            when lower(encounters.encounter_type) in (
+                 'emergency department'
+                ,'acute inpatient'
+            )
+            then 1
+            else 0
+          end as is_excluded
+    from observations_within_range
+    left join encounters
+        on observations_within_range.patient_id = encounters.patient_id
+        and observations_within_range.observation_date between 
+            encounters.encounter_start_date and encounters.encounter_end_date
+)
+
+, valid_observations as (
+
+    select
+        *
+    from observations_with_encounters
+    where is_excluded != 1
+
+)
+
+, systolic_bp_observations as (
+
+    select
+          patient_id
+        , observation_date
+        , result
+        , measure_id
+        , measure_name
+        , measure_version
+        , performance_period_begin
+        , performance_period_end
+        , row_number() over(partition by patient_id, observation_date order by observation_date desc, result asc) as rn
+    from valid_observations
+    where lower(normalized_description) = 'systolic blood pressure'
+
+)
+
+, diastolic_bp_observations as (
+
+    select
+          patient_id
+        , observation_date
+        , result
+        , row_number() over(partition by patient_id, observation_date order by observation_date desc, result asc) as rn
+    from valid_observations
+    where lower(normalized_description) = 'diastolic blood pressure'
+
+)
+
+, least_systolic_bp_per_day as (
+
+    select
+          patient_id
+        , observation_date
+        , result as systolic_bp
+        , measure_id
+        , measure_name
+        , measure_version
+        , performance_period_begin
+        , performance_period_end
+    from systolic_bp_observations
+    where rn = 1
+
+)
+
+, least_diastolic_bp_per_day as (
+
+    select
+          patient_id
+        , observation_date
+        , result as diastolic_bp
+    from systolic_bp_observations
+    where rn = 1
+
+)
+
+, patients_with_bp_readings as (
+
+    select
+          least_systolic_bp_per_day.*
+        , least_diastolic_bp_per_day.diastolic_bp
+    from least_systolic_bp_per_day
+    inner join least_diastolic_bp_per_day
+        on least_systolic_bp_per_day.patient_id = least_diastolic_bp_per_day.patient_id
+            and least_systolic_bp_per_day.observation_date = least_diastolic_bp_per_day.observation_date
+
+)
+
+, numerator as (
+
+    select
+          *
+        , case
+            when systolic_bp < 140 and diastolic_bp < 110 --needs to be changed back to 90; 110 for testing against synthetic data
```

Comment:
I will work on updating some observation values in the synthetic dataset tomorrow morning to help with this issue. Thanks for bringing it to my attention. 🙏